business_requirement_earned_value/README.rst

```diff
@@ -1,81 +1,43 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
-   :alt: License: AGPL-3
+=================================
```

Comment:
Please update the badge with latest template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst